organization/minutes-28-06-2021.md

```diff
@@ -0,0 +1,62 @@
+Observing: Jacco van der Weerd, Elena Findley-de Regt
+
+  
+
+## 1. Proposal: new board member
+
+The board of directors would like to ask the general assembly to appoint Eric Herman to the board of directors for an initial term of 1 year per Juli 1st, taking on the role of treasurer of the vereniging.  
+  
+To clearly separate the director role from the staff role he has his employment contract will be reduced by 2 hours per week.  
+  
+With this appointment, the board can resume its duties whilst it works on recruiting new high-profile board members. To facilitate this the board is forming a strategic council. Eric will focus on professionalization of the board.
+
+- *The group discussed the need for additional future board members, as well as desirable profiles*
+-   *The General Assembly agreed to the proposal*
+    
+## 2. Proposal: general assembly in the autumn
+
+The board of directors would propose to organize a second general assembly in the second half of the year.
+
+The goal of this second general assembly is to decide on an updated policy plan for 2022-2026.
+
+Alongside this, we would like this general assembly to have a program for a wider group of non-member stakeholders like public administrations, governments and other public collaborations to involve them in our future.
+
+-   *The General Assembly agreed to the proposal*
+    
+
+## 3. Policy plan 2022-2026
+
+The board of directors would like to invite the members to work with them on the updated policy plan for 2022-2026, which will succeed [our policy plan 2019-2022](https://about.publiccode.net/organization/policy-plan.html).
+
+
+-   *Eric will lead a working group to create the policy plan. Ben and Boris offered to support.*
+    
+
+## 4. Progress measures for philanthropists
+
+During the General Assembly, Ben mistakenly confirmed that the Board had adopted the targets previously proposed. A review of board minutes after this General Assembly clarified that the board did not adopt these targets at the February 2021 board meeting, nor at any subsequent board meetings.
+
+-  *An update was provided on targets set for philanthropic renewal.*
+-   *June 2021 progress against targets:*
+  -   *Target 1 (5 projects live and in production): 40% achieved*
+  -   *Target 2 (codebases or members in 3 countries): 33% achieved*
+  -   *Target 3 (coverage by one national and one international tier 1 publication): 0%*
+  -   *Target 4 (€100,000 funds publicly raised): €10,000 achieved*
+    
+
+## 5. Any other business
+
+-  *Pieter raised corrections to statements made during the April 2021 Foundation for Public Code community call about new board members:*
+  - *Pieter stepped down because he felt he didn’t have enough influence on the course of the Foundation*
+  -  *At the time Arnout resigned from the board, he had significant time available as he was no longer CTO of Adyen*
+    
+
```

Comment:
Are two blank lines needed?

Reply:
This was done in collaboration with Elena. We can double-check